### DISCUSSION TITLE
Keeshare logs

### DISCUSSION THREAD

**Discussion by brignolff** [keepassxreboot/keepassxc#10640] [Q&A] [answered]
Hi, 
I repeatedly saw some export errors appear when using keeshare with the shared .kdbx file hosted on a distant drive.
Some time it sync some it doesn't, some times it sync with the error message.
The only details that are appearing are saying that access was denied.

Are there logs somewhere that could help me find if the problem comes from Keepass or from our distant drive ?

Best regards

**Comment by droidmonkey** [comment, marked as answer]
If access is denied then you need to fix your permissions to the network storage location. Also ensure it is up and available.
